test/runtests.jl

```diff
@@ -4,10 +4,12 @@ using SBML
 using SBML: Model, Reaction, Species
 using Symbolics, Unitful
```

Comment:
Can we move `using Symbolics` inside the `symbolics.jl` file?  The rationale is that if I want to do quick tests I can simply comment out the `include` line :slightly_smiling_face:

Reply:
there are extra tests that depend on this. I'll make a condition there so that it can be switched off easily.